website/src/lib/icons.ts

```
@@ -0,0 +1,96 @@
+  download: {
+    path: `<path d="M12 3v12M7 10l5 5 5-5M5 21h14"/>`,
+  },
+  layers: {
+    path: `<path d="M12 3l9 5-9 5-9-5zM3 13l9 5 9-5M3 16.5l9 5 9-5"/>`,
+  },
+};
+
+export function getIconSvg(name: string, size: number = 24): string {
+  const icon = icons[name];
+  if (!icon) {
+    console.warn(`[icons.ts] Missing icon: "${name}"`);
```

ESLint

[error] 'console' is not defined.

(no-undef)